core/services/pipeline/task.bridge.go

```diff
@@ -36,6 +36,7 @@ type BridgeTask struct {
 var _ Task = (*BridgeTask)(nil)
 
 var zeroURL = new(url.URL)
+var stalenessCap = time.Duration(30 * time.Minute)
```

Comment:
Could this be a `const`?
```suggestion
const stalenessCap = time.Duration(30 * time.Minute)
```

Reply:
Done